docs/cloud-linter/index.md

```diff
@@ -56,6 +60,14 @@ DynamoDB:
 CloudWatch:
 - GetMetricData: 20 TPS
 
+S3:
+- ListBuckets
+- ListBucketMetricsConfigurations: 10 TPS
+
+API Gateway:
+- GetRestApis: 10 TPS
+- GetRestApi
```

Comment:
@pgautier404 I didn't see a limit on this call. Do you remember if that is done in a different way?

Reply:
@nand4011 The methods for processing those resources are using the metrics and control plane limiters in use throughout. Let me know if that doesn't address the question and we can take a look at it.